edit-post/README.md

```diff
@@ -0,0 +1,50 @@
+## Extending the post editor UI
+
+Extending the editor UI can be accomplished with the `registerPlugin` API, allowing you to define all your plugin's UI elements in one place.
+
+Refer to [the plugins module documentation](../plugins/) for more information.
+
+### Plugin Components
+
+The following components can be used with the `registerplugin` ([see documentation](../plugins)) API.
+They can be found in the global variable `wp.editPost` when defining `wp-edit-post` as a script dependency.
+
+Experimental components can be found under `wp.editPost.__experimental`. Experimental components are still being evaluated and can change in a future version.
+
+#### PluginSidebar (Experimental)
+
+Renders a sidebar when activated. The contents within the `PluginSidebar` will appear as content within the sidebar.
```

Comment:
Not totally clear to me what that means? How to make sure the sidebar is activated?